pkgs/development/python-modules/annoy/default.nix

```diff
@@ -18,12 +20,22 @@ buildPythonPackage rec {
     hash = "sha256-nL/r7+Cl+EPropxr5MhNYB9PQa1N7QSG8biMOwdznBU=";
   };
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail "'nose>=1.0'" ""
```

Comment:
Isn't pynose providing this?

Reply:
No sure, what the issue is as `pythonRemoveDeps` can also not handle it.